.github/actions/artifacts/action.yml

```diff
@@ -20,13 +24,22 @@ inputs:
 runs:
   using: 'composite'
   steps:
-    - name: Upload to codecov
-      uses: codecov/codecov-action@e0b68c6749509c5f83f984dd99a76a1c1a231044 # v4.0.1
-      with:
-        token: ${{ inputs.token }}
-        flags: ${{ inputs.type }}
-        files: ${{ inputs.files }}
-        override_commit: ${{ inputs.commit_sha}}
-        plugin: noop
-        verbose: true
+    - name: Download and Verify Codecov CLI
+      shell: bash
```

Comment:
Please use the `shell` seen in actions/setup-sentry to ensure that these commands are properly logged and properly stop on error:

`actions/setup-sentry/action.yml`
```
# the default for "bash" is:
#      bash --noprofile --norc -eo pipefail {0}
shell: bash --noprofile --norc -eo pipefail -ux {0}
```

This becomes unnecessary if you move all shell code to their own files (just don't forget to `set -euxo pipefail` on the first line).

Reply:
Resolved by `set -euo pipefail` in scripts, instead.